### PR TITLE
enhancement: Update auction registration flow UI

### PR DESCRIPTION
**Pull request by olerichter00**
The type of this PR is: Enhancement

This PR resolves [CX-1162]

### Description

This PR updates the auction registration flow to match the following design specs https://www.figma.com/file/DGgxE9lFvi8A9WRJkJqVjX/Nav-%2B-Modal-Headers?node-id=41%3A343

| Before | iOS |
|---|---|
|Simulator Screen Shot - iPhone 12 mini - 2021-05-10 at 13 49 32|image|
|---|---|
| iOS | iOS | 
|---|---|
|image|image|
|---|---|
|Android|---|
|---|---|
|image||

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [x] I have tested my changes on **iOS** and **Android**.
- [x] I have added tests for my changes, or my changes don't require testing, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/master/docs/adding_state_migrations.md))
- [x] I have added a [CHANGELOG.yml](/CHANGELOG.yml) entry or my changes do not require one.

[CX-434]: https://artsyproduct.atlassian.net/browse/CX-434
[CX-1161]: https://artsyproduct.atlassian.net/browse/CX-1161

[CX-1162]: https://artsyproduct.atlassian.net/browse/CX-1162

### CODE REVIEW

File: src/lib/Components/Bidding/Screens/__tests__/ConfirmBid-tests.tsx [hunk 1]
```diff
@@ -730,7 +730,7 @@ describe("ConfirmBid for unqualified user", () => {
 
     fillOutFormAndSubmit(component)
 
-    expect(component.root.findByType(Modal).findAllByType(Text)[1].props.children).toEqual(
+    expect(component.root.findByType(Modal).findAllByType(Sans)[1].props.children).toEqual(
```
Comment (pvinis) [inline src/lib/Components/Bidding/Screens/__tests__/ConfirmBid-tests.tsx:733]: How come these changed from `Text` to `Sans`, when you changed everything to `Text` 🤔?

**Comment by pvinis** [comment]
Other than the tests question, my reaction to this was "NIIIIIIIIIIIIIIIICE"!! I'm excited to get rid of old styles and get more consistent. One less screen! I only didn't review yet because I wanna check the test question.

**Comment by brainbicycle** [comment]
Is this good to go?

File: src/lib/Components/Bidding/Screens/__tests__/ConfirmBid-tests.tsx (hunk 1 above)
Comment (olerichter00) [reply src/lib/Components/Bidding/Screens/__tests__/ConfirmBid-tests.tsx:733]: In the `Modal` we still use `Sans`. `Text` was imported from `"react-native"` in this test but now we're importing it from `plalette` and that's why we can't use `Text` here anymore.

Comment (pvinis) [reply src/lib/Components/Bidding/Screens/__tests__/ConfirmBid-tests.tsx:733]: Cool. Thanks for the explanation!

**Comment by olerichter00** [comment]
> Is this good to go?

I just need to make a small adjustment, test it and then it'll be good to go.

File: CHANGELOG.yml [hunk 2]
```diff
@@ -15,12 +15,7 @@ upcoming:
     - Add artwork Material filter - dzmitry tratsiak
     - Prevent android dark mode forced on users - kizito
     - Fix unusable slider on android - dzmitry
-    - Add search UI for material filter - dzmitry tratsiak
-    - Implements custom size filter with localization (behind feature flag) - damon
-    - Adds artist nationality & ethnicity filter (behind feature flag) - damon
-    - Add order history to user profile (behind feature flag) - alexj105
-    - Enable lotsByFollowedArtists - ole
-    - Add ability to unselected "ways to buy" filter - dzmitry tratsiak
+    - Update auction registration flow UI - ole
```
Comment (pvinis) [inline CHANGELOG.yml:18]:
```suggestion
    - Update auction registration flow UI - ole
    - Add search UI for material filter - dzmitry tratsiak
    - Adds artist nationality & ethnicity filter (behind feature flag) - damon
    - Add order history to user profile (behind feature flag) - alexj105
    - Enable lotsByFollowedArtists - ole
    - Add ability to unselected "ways to buy" filter - dzmitry tratsiak
```

File: CHANGELOG.yml [hunk 3]
```diff
@@ -15,8 +15,8 @@ upcoming:
     - Add artwork Material filter - dzmitry tratsiak
     - Prevent android dark mode forced on users - kizito
     - Fix unusable slider on android - dzmitry
+    - Update auction registration flow UI - ole
     - Add search UI for material filter - dzmitry tratsiak
```
Comment (pvinis) [inline CHANGELOG.yml:19]:
```suggestion
    - Add search UI for material filter - dzmitry tratsiak
    - Implements custom size filter with localization (behind feature flag) - damon
```

**Pull request merged**